students/hyska_monika/lesson_02_flow_control/BishopMoves.py

```diff
@@ -0,0 +1,17 @@
+# The program print that Bishop can move from one field to second field
+import math
+
+v1 = int(input("Put vertical position for 1st field (from 1 to 8): "))
+h1 = int(input("Put horizontal position for 1st field(from 1 to 8): "))
+v2 = int(input("Put vertical position for 2nd field(from 1 to 8): "))
+h2 = int(input("Put horizontal position for 2nd field(from 1 to 8): "))
+
+vector_1 = int(math.fabs(v1 - v2))
```

Comment:
There is a built-in function `abs()`. This one is supposed to be used on `float`s and it may fail with bigger numbers:
```
$ ipython3
In [2]: int(0.9999)
Out[2]: 0
```